cmd/mt/detect/resolve.go

```diff
@@ -0,0 +1,75 @@
+package detect
+
+import (
+	"errors"
+	"os"
+
+	mtcmd "github.com/Smartling/smartling-cli/cmd/mt"
+	"github.com/Smartling/smartling-cli/services/helpers/env"
+
+	api "github.com/Smartling/api-sdk-go/api/mt"
+	"github.com/spf13/cobra"
+)
+
+func resolveFileType(cmd *cobra.Command) string {
+	if cmd.Flags().Changed(fileTypeFlag) {
+		return fileType
+	}
+	envVarName := env.VarNameFromCLIFlagName(fileTypeFlag)
+	if val, isSet := os.LookupEnv(envVarName); isSet {
+		return val
+	}
+	return fileType
+}
+
+func resolveOutputTemplate(cmd *cobra.Command, fileConfig mtcmd.FileConfig) string {
+	if cmd.Flags().Changed(outputTemplateFlag) {
+		return outputTemplate
+	}
+	envVarName := env.VarNameFromCLIFlagName(outputTemplateFlag)
+	if val, isSet := os.LookupEnv(envVarName); isSet {
+		return val
+	}
+	if fileConfig.MT.FileFormat != nil {
+		return *fileConfig.MT.FileFormat
+	}
+	return outputTemplate
+}
+
+func resolveAccountUID(cmd *cobra.Command, cnfAccountID string) (api.AccountUID, error) {
```

Comment:
I believe this and similar "resolve" functions should be in one place and reused by all commands. The `xxx/resolve.go` files must contain something very specific, and I anticipate in most cases will be empty.